package.json

```
@@ -0,0 +1,36 @@
+{
+  "name": "frames-examples-pnpm",
```

```suggestion
  "name": "@dscvr-one/frames-examples",
```

Ref: https://dscvr.atlassian.net/browse/DCP-1591